### PR TITLE
Huha snapshot cleanup

### PR DESCRIPTION
**Pull request by shundhammer**
https://trello.com/c/YDDWzV5X

### CODE REVIEW

File: library/system/src/lib/yast2/fs_snapshot.rb [hunk 1]
```diff
@@ -65,6 +65,9 @@ class FsSnapshot
     LIST_SNAPSHOTS_CMD = "LANG=en_US.UTF-8 /usr/bin/snapper --no-dbus --root=%{root} list"
     VALID_LINE_REGEX = /\A\w+\s+\| \d+/
 
+    # Predefined snapshot cleanup strategies (the user can define custom ones, too)
```
Comment (jreidinger) [inline library/system/src/lib/yast2/fs_snapshot.rb]: I do not see how user can use his own defined one

Comment (shundhammer) [inline library/system/src/lib/yast2/fs_snapshot.rb]: Not in the context of this Ruby class, but in general, he can. Snapper comes with those predefined ones you see here, but a user can write his own scripts and make snapper use them on request. Arvin knows more details about that.

File: library/system/test/fs_snapshot_test.rb [hunk 2]
```diff
@@ -118,6 +121,46 @@ def logger
           expect(snapshot).to be(dummy_snapshot)
         end
       end
+
+      context "when a cleanup strategy is set" do
+        let(:output) { { "stdout" => "2", "exit" => 0 } }
+        let(:dummy_snapshot) { double("snapshot") }
+        let(:snapshot_command) { CREATE_SINGLE_SNAPSHOT + OPTION_CLEANUP_NUMBER }
+
+        it "returns the created snapshot" do
```
Comment (jreidinger) [inline library/system/test/fs_snapshot_test.rb]: when you run `rspec --format doc` it sounds bit strange, that all when have same result. More important here is to mention that strategy is used like 
```
it "creates snapshot with given strategy and returns the created snapshot" do
 ...
end
```
and similar below, so it is obvious how cleanup strategy affect behavior.

File: library/system/src/lib/yast2/fs_snapshot.rb (hunk 1 above)
Comment (jreidinger) [inline library/system/src/lib/yast2/fs_snapshot.rb]: Ah a bit confusing as I think it mean, that you can pass to parameter also other strategies, which do not work.

**Pull request merged**